extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -0,0 +1,472 @@
+   which slows the page load.
+
+1. Media: Audio must not enable autoplay.  This includes both the `<amp-audio>`
+   tag as well as all audio-including video tags, as described in the previous 
+   point.
+
+   _Rationale_: Same as for video.
+
+1. Analytics: `<amp-analytics>` viewability tracking may only target the full-ad
+   selector, via  `"visibilitySpec": { "selector": "amp-ad" }`, as defined in
+   [Issue #4018](https://github.com/ampproject/amphtml/issues/4018) and
+   [PR #4368](https://github.com/ampproject/amphtml/pull/4368).  In
+   particular, it may not target any selectors for elements within the ad 
+   creative.
+
+   _Rationale_: In some cases, A4A may choose to render an ad creative in an 
+   iframe.  In those cases, host page analytics can only target the entire 
+   iframe anyway, and won’t have access to any finer-grained selectors.
+
+   _Example_:
+  
+  ```html
+<amp-analytics id="nestedAnalytics">
+  <script type="application/json">
+  {
+    "requests": {
+       "visibility": "https://example.com/nestedAmpAnalytics"
+    },
+    "triggers": {
+      "visibilitySpec": {
+        "selector": "amp-ad",
+        "visiblePercentageMin": 50,
+        "continuousTimeMin": 1000
+      }
+    }
+  }
+  </script>
+</amp-analytics>
+  ```
+  
+  _This configuration sends a request to URL
+  `https://example.com/nestedAmpAnalytics` when 50% of the enclosing ad has been
+  continuously visible on the screen for 1 second._
+
+### Boilerplate
+
+A4A creatives require a different, and considerably simpler, boilerplate style line than
+[general AMP documents do](https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md):
+
+```html
+<style amp-a4a-boilerplate>body{visibility:hidden}</style>
+```
+
+_Rationale:_ The `amp-boilerplate` style hides body content until the AMP 
+runtime is ready and can unhide it.  If Javascript is disabled or the AMP 
+runtime fails to load, the default boilerplate ensures that the content is 
+eventually displayed regardless.  In A4A, however, if Javascript is entirely 
+disabled, A4A won't run and no ad will ever be shown, so there is no need for
+the `<noscript>` section.  In the absence of the AMP runtime, most of the 
+machinery that A4A and ad creatives rely on (e.g., analytics for visibility 
+tracking or `amp-img` for content display) won't be available, so it's better to 
+display no ad than a malfunctioning one.
+
+Finally, the A4A boilerplate uses `amp-a4a-boilerplate` rather than
+`amp-boilerplate` so that validators can easily identify it and produce
+more accurate error messages to help developers.
+
+Note that the same rules about mutations to the boilerplate text apply as in 
+the [general AMP boilerplate](https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md).
+
+### CSS
+
+1. `position:fixed` and `position:sticky` are prohibited in creative CSS.
+
+   _Rationale_: position:fixed breaks out of shadow DOM, which A4A depends on.
```

Comment:
Just a note: @dvoytenko did some experiment with pain containment that would possibly allow adding this in the future.